### ISSUE TITLE
[23.0.2] composer install giving error

### ISSUE THREAD

**Issue by jesscall** [aces/Loris#7172]
**Describe the bug**

```
lorisadmin@ccna-dev:/var/www/ccna$ composer install
Loading composer repositories with package information
Warning from https://packagist.org: You are using an outdated version of Composer. Composer 2.0 is now available and you should upgrade. See https://getcomposer.org/2
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for phpunit/phpunit 7.0.0 -> satisfiable by phpunit/phpunit[7.0.0].
    - Conclusion: don't install sebastian/comparator 2.1.3
    - phpunit/phpunit 7.0.0 requires sebastian/comparator ^2.1 -> satisfiable by sebastian/comparator[2.1.0, 2.1.1, 2.1.2, 2.1.3].
    - sebastian/comparator 2.1.0 requires sebastian/diff ^2.0 -> satisfiable by sebastian/diff[2.0.1].
    - sebastian/comparator 2.1.1 requires sebastian/diff ^2.0 -> satisfiable by sebastian/diff[2.0.1].
    - sebastian/comparator 2.1.2 requires sebastian/diff ^2.0 -> satisfiable by sebastian/diff[2.0.1].
    - Conclusion: don't install sebastian/diff 2.0.1|install sebastian/comparator 2.1.3
```

**Server Environment (if known):**
- LORIS Version: 23.0.2
- Linux distribution and Version: CentOS 7
- Composer version: 1.6.2

_____
Error is resolved when changing the following line in `composer.json`
```
"phpunit/phpunit" : "7.0.0",
```

to:

```
"phpunit/phpunit" : "^7.0.0",
```

**Comment by jesscall** [comment]
@driusan Rida said you would know more about this

**Comment by driusan** [comment]
I don't know anything about this on 23.0.x. phpunit is upgraded to 9.4 on main, but that shouldn't affect the 23.0 branch. Did you upgrade or change your version of PHP?

**Comment by jesscall** [comment]
yes, on php 7.3

**Comment by driusan** [comment]
This will be fixed by #7222 if it ever CI..

**Comment by driusan** [comment]
PR was merged
